### DISCUSSION TITLE
How to implement saveMissing functionality?

### DISCUSSION THREAD

**Discussion by simpautus** [sergiodxa/remix-i18next#55] [Q&A] [answered]
Is it possible to save missing translation keys?

**Comment by sergiodxa** [comment, marked as answer]
The latest version (v4.0.0) uses normal backends and that should let you define the saveMissing option in i18next itself.

**Reply by simpautus** [reply]
Thank you! Works great.

**Reply by duerdod** [reply]
Sorry to bring this old one up. Did you ever have an issue with 405 'Method not allowed' when backend is trying to POST missing translations to file?

**Reply by simpautus** [reply]
> Sorry to bring this old one up. Did you ever have an issue with 405 'Method not allowed' when backend is trying to POST missing translations to file?

I haven't been on this for a while but I think I ended up placing the translation files in public folder for some reason. Also the addPath was to an api route with exported action function to handle the posted missing translation.
